### DISCUSSION TITLE
show info from command line if app is apple silicon

### DISCUSSION THREAD

**Discussion by KwisatzJim** [Homebrew/discussions#2789] [Everyday usage]
Hi, been searching for a way to do this and haven't found it, so thought I'd try here.  I'm wondering if there's a way from the command line to get info about an app and have it show if it's Apple Silicon native?  
Right now if I run "brew info bash" it shows brief information and a few analytics. Using -v gives more analytics.  Would like something that shows what the website shows i.e. : 

Intel | Monterey | ✅
          Big Sur | ✅
          Catalina | ✅
          64-Bit Linux | ✅

Apple Silicon | Monterey | ✅
                          Big Sur | ✅

**Comment by carlocab** [comment]
Try `brew info`. Here's a formula that has no Apple Silicon bottle:
```
❯ brew info winexe
winexe: stable 1.00
[snip]
```
Here's one that does:
```
❯ brew info hello
hello: stable 2.11 (bottled)
[snip]
```

**Reply by KwisatzJim** [reply]
so if it says (bottled) that means it's Apple Silicon?

**Reply by SMillerDev** [reply]
If you run it on Apple Silicon and it says `bottled`, yes

**Reply by KwisatzJim** [reply]
Gotcha. That makes it super easy to identify. Thanks 👍🏼👍🏼

**Comment by gromgit** [comment]
A quick one-liner that should work on all supported Homebrew platforms, even Linux:
```
brew info --json <formula> | grep -q arm64 && echo Apple Silicon native
```

**Reply by KwisatzJim** [reply]
ok cool.  I had seen the json options in the man pages but I don't understand json parsing very well so couldn't figure out how to get the data I wanted.  thanks for this!  I'll keep it in my tool box 👍🏻👍🏻👍🏻👍🏻

**Reply by carlocab** [reply]
Try
```
brew info --json=v2 hello | jq --raw-output '.formulae[].bottle.stable.files | keys | .[]'
```

**Reply by KwisatzJim** [reply]
Oh I like this one. Tells you all the supported architecture and OS like the website. Perfect 👍🏼👍🏼👍🏼👍🏼
